src/main/java/uk/gov/justice/digital/nomis/api/UserDetail.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.nomis.api;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Builder
+public class UserDetail {
```

Comment:
Can you explain what a UserDetail is? It looks like a collection of caseloads?

Reply:
UserDetail renamed to StaffUser - Its a member of staff, I've now added their name and currrent active status